docs/cli/powershell.rst

```diff
@@ -1,7 +1,7 @@
 Package Manager Console (Visual Studio)
 =======================================
 
```

Comment:
The known issues from CLI also apply to PMC commands for ASP.NET Core and .NET Core applications... we should tell folks that for those project types we just invoke the CLI commands, and link to the CLI known issues section.